### PR TITLE
Add formatting option to align property value pairs

### LINKED ISSUE #672: Add code formatting option for lining up assignment operators in DSC configurations
When a user writes a DSC configuration, they might line up their assignment statements like this:

```powershell
configuration MyConfiguration {
    xDscWebService PSDSCPullServer {
        Ensure                = “Present”
        EndpointName          = “PSDSCPullServer”
        Port                  = 8080
        PhysicalPath          = “$env:SystemDrive\inetpub\wwwroot\PSDSCPullServer”
        CertificateThumbPrint = “AllowUnencryptedTraffic”
    }
}
```

We should do this formatting for them so that when they add new properties they can reformat the configuration and have all the properties be lined up at the equal signs again.

/cc @jpsnover @smurawski

### PR DESCRIPTION
**Pull request by kapilmb**
Resolves #672 
Related PR PowerShell/PSScriptAnalyzer#753

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -408,6 +408,11 @@
           "default": true,
           "description": "Does not reformat one-line code blocks, such as \"if (...) {...} else {...}\"."
         },
+        "powershell.codeFormatting.alignPropertyValuePairs": {
+          "type": "boolean",
+          "default": true,
+          "description": "Align assignment statements in a hashtable."
```
Comment (daviwil) [inline package.json]: I'd expand this description to include "or a DSC configuration"

Comment (kapilmb) [inline package.json]: fixed!

**Comment by daviwil** [comment]
:shipit:

**Comment by kapilmb** [comment]
Thanks @daviwil !

**Pull request merged**